tests/README.rst

```diff
@@ -0,0 +1,74 @@
+===========
+Don't Panic
+===========
+
+The Hypothesis test suite is large, but we've written these notes to help you
+out.  It's aimed at contributors (new and old!) who know they need to add tests
+*somewhere*, but aren't sure where - or maybe need some hints on what kinds of
+tests might be useful.  Others might just be interested in how a testing
+library tests itself!
+
+
+The very short version
+======================
+
+- To improve code coverage (eg because ``make check-coverage`` / the Travis
+  build is failing), go to ``cover/``
+- For longer / system / integration tests, look in ``nocover/``
+- For tests that require an optional dependency, look in the appropriate
+  directory
+
+In each case there will be further notes to help you.
+
+
+About each group of tests
+=========================
+
+Still here?  Here's a note on what to expect in each directory.
+
+``common/``
+    Useful shared testing code, including test setup and a few helper
+    functions in ``utils.py``.  Also read up on pytest features such as
+    ``mark.parametrize``, ``mark.skipif``, and ``raises``.
+
+``cover/``
+    The home of enough tests to get 100% branch coverage - you'll probably
+    spend most of your time here.  Ideally this would be as quick as possible
+    while remaining comprehensive.  More details inside.
+
+``datetime/``
+    *Tests for the deprecated* ``hypothesis.extra.datetime`` *module, which
+    depends on the ``pytz`` package.*
```

Comment:
Something funky is happening with the rendering of `pytz`.

![screen shot 2017-11-26 at 12 07 17](https://user-images.githubusercontent.com/301220/33239817-5de5fb58-d2a2-11e7-9664-ebe10abf0800.png)